### DISCUSSION TITLE
Syntax Error in main_test.py

### DISCUSSION THREAD

**Discussion by derekstrand0** [DIGITALCRIMINALS/OnlyFans#144] [Q&A]
I am getting the following syntax error after attempting to run start_ofd.py for the first time.

![image](https://user-images.githubusercontent.com/95518965/144699861-c7767270-ec54-4782-bb10-0953ec722226.png)

For reference, here's the line the error refers to:
![image](https://user-images.githubusercontent.com/95518965/144699889-adfc0368-ac41-472a-a01e-81ea757027ac.png)

I haven't used Python in years so I have no idea how to fix this. I would appreciate some assistance.

**Comment by Softload3r** [comment]
Try updating your requirements: 
`python3.10 -m pip install --upgrade -r requirements.txt
`

And check if you get an error while updating the requirements. For me I got an error which said that I needed Microsoft Visual C++ 14.0 or greater

Here's a pic with what I have installed:
https://imgur.com/a/xxEIbbK

Just download it here:
https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16

**Reply by sinkust** [reply]
I have the same problem, has anyone found a solution yet?

**Reply by DIGITALCRIMINAL** [reply]
Use Python 3.10
